server/plugin/taskLinkPreview.go

```diff
@@ -159,3 +159,44 @@ func (p *Plugin) PostBuildDetailsPreview(linkData []string, link, userID, channe
 	model.ParseSlackAttachment(post, []*model.SlackAttachment{attachment})
 	return post, ""
 }
+
+func (p *Plugin) PostReleaseDetailsPreview(linkData []string, link, userID, channelID string) (*model.Post, string) {
+	organization := linkData[3]
+	project := linkData[4]
+	releaseID := strings.Split(linkData[5], "&")[1][10:]
+	releaseDetails, _, err := p.Client.GetReleaseDetails(organization, project, releaseID, userID)
+	if err != nil {
+		return nil, ""
+	}
+
+	post := &model.Post{
+		UserId:    userID,
+		ChannelId: channelID,
+	}
+
+	environments := p.getPipelineReleaseEnvironmentList(releaseDetails.Environments)
+	attachment := &model.SlackAttachment{
+		AuthorName: "Azure Pipeline",
+		AuthorIcon: fmt.Sprintf("%s/plugins/%s/static/%s", p.GetSiteURL(), constants.PluginID, constants.FileNamePipelinesIcon),
+		Title:      fmt.Sprintf(constants.PipelineDetailsTitle, releaseDetails.Name, releaseDetails.Link.Web.Href, releaseDetails.ReleaseDefinition.Name),
```

Comment:
Why aren't we using the `TitleLink` property of the slack attachment?

Reply:
I was not aware of it and in many places this was already used by other devs, taking a note we can do it after merging everything.